### ISSUE TITLE
Improve look&feel of external link warning in Kiwix Server

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-tools#711]
In Kiwix Server, when a user is about to leave the pages serverd from the ZIM file, then they click on a so called "external link". That means a link which is not served by Kiwix Server, but by a Web server... on the Web.

This is a very normal situation as ZIM file pages have a lot of external links. But of course we want to inform the user about that. This is a matter of transparency and to some extend of security.

Today such a page looks like this:
![Image](https://github.com/user-attachments/assets/0293b743-b462-43dd-8a4c-e737e4385fe3)

So basically the page has two functions:
* Inform the user
* Allow the user - with just a click - to jump to the external content.

The current design is really not nice, we would like to have something really better.

**Comment by benoit74** [comment]
To me there is more than just a design issue, but also wording issues.

Here we want to inform the user he just clicked a link which is not a resource from the ZIM but somewhere outside. The title "External link blocked" convey a meaning of "you shall not pass" where we just want to say something like "Be carefull, you are leaving the ZIM and going to the Internet, are you sure you want to do that?". I think we should rework that as well.

In addition to this warning, in iFixit I felt like it was important to indicate to the user why this link which has just been clicked is not inside the ZIM. In quite a lot of situations, the user might expect the link he just clicked to be available inside the ZIM. In iFixit we have multiple reasons for such a link to go to the internet:
- `unavailable_offline`: content which is impossible to make available offline (technical limitation usually)
- `not_yet_available`: content which could me made available offline, but scraper is not yet capable to do it, we need money / support / contributions to make it work (goal here is to convey interest on helping us or at least give some incitation on saying "we would really need you to include this kind of content)
- `not_scrapped`: content specifically excluded from this ZIM: while the scraper could technically include this content, the ZIM publisher decided to not include it
- `missing`: this content was not available online when the scraper created the ZIM, so it cannot be available in the ZIM (typically the case where an online link leads to a 404) ; maybe it was just a temporary issue so it might be worth to go online and check if it has been fixed
- `error`: this content should be available, but for whatever reason the scraper failed to retrieve it / store it in the ZIM
- `external_content`: this link goes outside of the ZIM content, e.g. for ifixit we have links going to many other web domains, these links are external to what we want to put inside the ZIM, and they will always be, and it is really normal.

I don't know exactly yet how we can implement this technically speaking, but I would like to have designs / wordings for all these situations.

**Comment by siemsie** [comment]
Bouncing off Benoit's message – is there a clear answer as to _what_ goes on the page so that I have an idea of how to move forward?

**Comment by rgaudin** [comment]
Fix in https://github.com/kiwix/libkiwix/pull/1178
